content/en/docs/userguide/kiali.md

```diff
@@ -0,0 +1,70 @@
+---
+draft: false
+linktitle: Use Kiali to visualize traffic graph under Kmesh
+menu:
+  docs:
+    parent: user guide
+    weight: 19
+title: Use Kiali to visualize traffic graph under Kmesh
+toc: true
+type: docs
+
+---
+
+### Preparation
+
+1. Make default namespace managed by Kmesh.
+
+2. Deploy bookinfo as sample application and sleep as curl client.
+
+3. \[optional\] Install service granularity waypoint for service `reviews`.
+
+*The above steps could refer to [Install Waypoint | Kmesh](https://kmesh.net/en/docs/userguide/install_waypoint/#preparation). When installing Istio, we recommend installing Istio ambient mode instead of only installing Istiod, because Kiali currently depends on Istio components to work.*
```

Comment:
```suggestion
*The above steps could refer to [Install Waypoint | Kmesh](https://kmesh.net/en/docs/userguide/install_waypoint/#preparation). When installing Istio, we recommend installing [Istio ambient mode](https://istio.io/latest/docs/ambient/install/istioctl/) instead of only installing Istiod, because Kiali currently depends on Istio components to work.*
```